championship_manager_test.go

```diff
@@ -130,18 +130,26 @@ func (d dummyNotificationManager) SendRaceWeekendReminderMessage(raceWeekend *Ra
 	return nil
 }
 
-func (d dummyNotificationManager) SendMessage(msg string) error {
+func (d dummyNotificationManager) SendMessage(title string, msg string) error {
 	return nil
 }
 
-func (d dummyNotificationManager) SendMessageWithLink(msg string, linkText string, link *url.URL) error {
+func (d dummyNotificationManager) SendMessageWithLink(title string, msg string, linkText string, link *url.URL) error {
 	return nil
 }
 
 func (d dummyNotificationManager) SendRaceStartMessage(config ServerConfig, event RaceEvent) error {
 	return nil
 }
 
+func (d NotificationManager) GetCarList(cars string) string {
```

Comment:
this should be declared on dummyNotificationManager not NotificationManager - that's what's causing the build failure I think.

Reply:
Ooops.  I added those as an afterthought, after I did a test build.  That'll teach me not to change things without another test build.